src/pure-client.ts

```diff
@@ -0,0 +1,132 @@
+import axios, { AxiosRequestConfig, AxiosResponse } from 'axios'
+
+enum PureClientHeadersContentType {
+    APPLICATION_JSON = 'application/json'
+}
+
+interface PureClientHeaders {
+    'Content-Type': PureClientHeadersContentType
+    Accept: PureClientHeadersContentType
```

Comment:
So to refine my comment - the naming of this constant is probably confusing - as it is both the acceptable response body `Accept` and the way the body of the request is being made `Content-Type` - which really only needs to be incuded in the `post` and `put` methods. So probably just need to split the enum something like;
```
enum PureClientHeadersContentType {
    MEDIA_TYPES = 'application/json, application/xml'
}

enum PureClientHeadersAcceptTypes {
    MEDIA_TYPES = 'application/json, application/ld+json'
}
```

```suggestion
    Accept: PureClientHeadersContentType
```

Reply:
I agree, will do (y)